synapse/rest/client/account.py

```diff
@@ -518,6 +525,7 @@ async def on_POST(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
         threepid_send_requests.labels(type="msisdn", reason="add_threepid").observe(
             send_attempt
         )
+        logger.info("MSISDN %s is already in use by %s", msisdn, existing_user_id)
```

Comment:
This doesn't look like it's in the right place?

Reply:
no, looks like a copy paste fail. thanks.